mbuild/packing.py

```diff
@@ -52,20 +53,24 @@ def fill_box(compound, n_compounds=None, box=None, density=None, overlap=0.2, se
     If `box` and `density` are not None, the corresponding number of
     compounds will be calculated internally.
 
+    For the cases in which `box` is not specified but generated internally,
+    the default behavior is to calculate a cubic box. Optionally,
+    `aspect_ratio` can be passed to generate a non-cubic box.
+
     Parameters
     ----------
     compound : mb.Compound or list of mb.Compound
     n_compounds : int or list of int
     box : mb.Box
+    aspect_ratio : list of float
     overlap : float, units nm
+    edge : float, units nm
     density : float, units kg/m^3
```

Comment:
Could you add descriptions for each of these arguments in the docstring (I know several of these existed already without a description)?